src/components/notification/NotificationView.tsx

```diff
@@ -98,6 +101,21 @@ const NotificationViewComponent = (props: Props) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [detectedTokensRedux, handleBackupPress, theme])
 
+  // Check for 2fa/otp notifications
+  useAsyncEffect(
+    async () => {
+      const otpReminderModal = await getOtpReminderModal(account)
+
+      if (otpReminderModal != null) {
+        setOtpReminderCard(
+          <NotificationCard type="warning" title={lstrings.otp_reset_modal_header} message={lstrings.notif_otp_message} onPress={otpReminderModal} />
```

Comment:
This was a plot twist! I was expecting you to save the `otpReminderModal` function in the `React.useState`, not the whole JSX for the card. Then, if you have an OTP modal callback function, you would render the card normally like the others.

Storing JSX in state is super-weird, but I do see that it works, so no problem. Just really suprising.

Reply:
Yeah I was following the pattern for the above effect